0800-appendix-197378-trib-to-owen.Rmd

```diff
@@ -22,7 +22,7 @@ my_site <- 197378
 ## Site Location {.unnumbered}
 
 PSCIS crossing `r as.character(my_site)` is located on
-`r fpr::fpr_my_pscis_info()`, on `r fpr_my_pscis_info(col_pull = road_name)`. The site was located approximately 1km upstream from the confluence with Owen Creek, which drains the nearby Owen Lake. This crossing is the responsibility of West Fraser Mills Ltd.
+`r fpr::fpr_my_pscis_info()`, on `r fpr_my_pscis_info(col_pull = road_name)` road. The site was located approximately 1km upstream from the confluence with Owen Creek, which drains the nearby Owen Lake. The road at this location is owned by YLP Woodlands GP Ltd.
```

Comment:
Road is owned by MoF.  YLP has "tenure" over the road in that they are allowed to use it and "can" fix it. I'm still a little unclear on it all but I do know they don't own it.  I think they basically rent it for free but need to keep it up. We shoudl do our homework and get our heads around it as best as possible and update our language accordingly.

Reply:
this is helpful 
https://www2.gov.bc.ca/gov/content/industry/forestry/forest-tenures/timber-harvesting-rights/road-permit-forestry

Reply:
okay interesting that makes more sense thanks.